NOTES.md

```diff
@@ -1,3 +1,32 @@
+# Release 2.2.1 (Modern Times)
+
+Released on 2015-11-16
+
+Bug fix release
+
+### Compatibility ###
+
+RethinkDB 2.2.1 is fully compatible with RethinkDB 2.2.0. Please read the
+[RethinkDB 2.2.0 release notes][release-notes-2.2.0] if you're upgrading from an
+older version.
+
+[release-notes-2.2.0]: https://github.com/rethinkdb/rethinkdb/releases/tag/v2.2.0
+
+### Bug fixes ###
+
+* Fixed a crash with the message "Guarantee failed: [found_hash_pair]" when running
+  `getAll` queries (#5085)
+* `rethinkdb export` and `rethinkdb dump` now limit the number of subprocesses to
+  reduce memory consumption (#4809)
+* Fixed a segmentation fault in  `orderBy.limit` changefeeds (#5081)
+* Fixed a crash when using `getAll` with unprintable keys (#5086)
```

Comment:
@danielmewes -- that description seems fine.  You could also say "illegal keys" or "unserializable keys" since users probably don't know what unprintable keys are.